XamlControlsGallery/PageHeader.xaml

```diff
@@ -98,7 +63,7 @@
                     TextTrimming="CharacterEllipsis"
                     TextWrapping="NoWrap" />
             </Grid>
-            <Border x:Name="commandBarBorder" Grid.Column="1">
+            <Border x:Name="commandBarBorder" Grid.Column="1" VerticalAlignment="Bottom">
```

Comment:
It looks like the header text is very indented. Expected? 

![image](https://user-images.githubusercontent.com/25991996/65173216-362a2700-da03-11e9-98d5-bf795396e4c8.png)

Reply:
The problem originates in the NavigationView which creates a left padding which is as wide as the back button and the toggle pane button so the header does not overlap with the two buttons.

See [this line in NavigationView.cpp](https://github.com/chingucoding/microsoft-ui-xaml/blob/b7984c3f76ad3598493e5cf9e9cad23ce71bacd8/dev/NavigationView/NavigationView.cpp#L3188).

Reply:
Hmm! I think this well and good when the content is supposed to be inline with the buttons (ie. where the "XAML Controls Gallery" text is) but not so great when the title is below (ie. the "Acrylic" text). 

What if we used @YuliKl's favorite tool: negative margins? 😜

Reply:
Haha, a negative margin would indeed fix this. But it would create a new problem:
The left padding is only present in minimal mode. So we would have to listen for that or otherwise the text starts clipping into the navigation pane of the NavigationView.

Setting the correct negative margin and not clearing it for non minimal displaymodes would result in this behaviour:
![navview-header-alignbug](https://user-images.githubusercontent.com/16122379/65220995-510ca200-dabc-11e9-844e-18f8a7e95c3b.gif)

So it would be a bit more work to fix that behavior, but with the beloved negative margin it would be possible 😅

Reply:
You're highlighting the thing I dislike most about the Minimal mode layout right now. I need more time to work on it, and would prefer not to block this PR on that change.
So: yes, expected; no, not pretty.